js/packages/components/shared-components/StickyMusicPlayer.tsx

```diff
@@ -6,15 +6,19 @@ import { useSafeAreaInsets } from 'react-native-safe-area-context'
 import { useStyles } from '@berty-tech/styles'
 import { useMusicPlayer } from '@berty-tech/music-player'
 import { useThemeColor } from '@berty-tech/store/hooks'
+import { navigationRef } from '@berty-tech/navigation'
 
 export const HEIGHT_OF_PLAYER = 60
 
+const hidePlayerRoutes: string[] = ['Chat.OneToOne', 'Chat.MultiMember', 'Chat.Group']
```

Comment:
```suggestion
const hidePlayerRoutes: (keyof ScreenParams)[] = ['Chat.OneToOne', 'Chat.MultiMember', 'Chat.Group']
```
`ScreenParams` comes from `@berty-tech/navigation`

also maybe add
`export type ScreenNames = keyof ScreenParams` in `navigation/types`

Reply:
Sure!